modules/ding_mobilesearch/includes/trigger.inc

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/**
+ * @file
+ * Stores a triggers for exporting content.
+ */
+
+/**
+ * Implements hook_menu_link_insert().
+ */
+function ding_mobilesearch_menu_link_insert($link) {
```

Comment:
The menu insert, update and delete has the same code. Could this be refactored into at _ding_mobilesearch_menu_link($link, $op = 'insert') ?